cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/CertificateRenewalTest.java

```diff
@@ -169,11 +166,12 @@ public static void after() {
                 } else {
                     reconcileCasComplete.fail(ar.cause());
                 }
-                async.flag();
+                context.completeNow();
             });
         return reconcileCasComplete.future();
     }
 
+    @SuppressWarnings("ResultOfMethodCallIgnored")
```

Comment:
Intellij flags certain IO method calls where  the result (that indicates success or failure) is ignored ,as warnings, in this case 

```
            clusterCaKeyFile.delete();
            clusterCaCertFile.delete();
            clusterCaStoreFile.delete();
```

I know not everyone uses IDEA,  but I want to reduce warning noise, so that we can focus on warnings that are significant.